### ISSUE TITLE
Tooltip cut off in Widget settings

### ISSUE THREAD

**Issue by RV-Adrian** [Rise-Vision/widget-text#124]
How to recreate:
Click on 'PUD Failover' tooltip icon in Text Widget settings

![image](https://user-images.githubusercontent.com/11826611/41657196-933dfae8-7460-11e8-8d07-b6fb4e596561.png)

The Problem:
Tooltip text leaks into iframe bounds

How It Should Work:
Tooltip should open on the left side instead, which should be a client-side setting

**Comment by pcsandford** [comment]
@RV-Adrian we have released a fix for this issue and it can be validated in this presentation: https://apps.risevision.com/editor/workspace/f77be6b2-a124-4df1-aef9-0379a9bcb3ea/?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f

We found that changing the tooltip opening to "left" could hide the trigger button, so we added CSS to reduce the size of the popup and keep it within the bounds of the modal. This gives it a similar size to the same popup in the calendar widget as it looks like the issue was caused with some changes to versions of angular bootstrap. Thanks!
